### DISCUSSION TITLE
How to get all Github Achievements

### DISCUSSION THREAD

**Discussion by younes-erraji** [community/community#106789] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

# Github Achievements

### I am just wondering how to get all of Github Achievements

> and I saw previously that those Github Achievements won't be available the next year so i am just wondering if this is true

**Comment by queenofcorgis** [comment, marked as answer]
Hi @younes-erraji, 

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team

Also know: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers.
